packages/react-form-renderer/src/files/use-field-api.js

```diff
@@ -96,7 +96,13 @@ const useFieldApi = ({ name, initializeOnMount, component, render, validate, res
         arrayValidator: calculateArrayValidator(props, validate, component, validatorMapper)
       });
     }
-  }, [validate, component, props.dataType]);
+    /**
+     * We have to stringify the validate array in order to preven infinite looping when validate was passed directly to useFieldApi
```

Comment:
I would convert this comment into a test case.

Reply:
I am not sure how would we test that something is not causing an infinite loop.

I will add a probably useless test that the hook won't crash after we pass the validate prop directly. And maybe change it. But that probably all we can do :shrug:

Reply:
Yeah, just check it works. (Otherwise it would probably crash the whole testing because of full memory.)